src/main/cpp/FfmpegPipelineFactory.cpp

```diff
@@ -8,15 +8,14 @@ FfmpegPipelineFactory::~FfmpegPipelineFactory()
 
 uint32_t FfmpegPipelineFactory::CreatePlayerPipeline(CPipelineOptions *pOptions, CPipeline **ppPipeline) {
 
-	if (NULL == pOptions->GetStreamData())
-		return ERROR_BASE_PIPELINE;
+	//if (NULL == pOptions->GetStreamData())
+	//	return ERROR_BASE_PIPELINE;
 	
 	*ppPipeline = NULL;
 	if (pOptions->GetStreamData())
```

Comment:
Do you have the logic inverted?

I thought if getStreamData is set then we would like to use the playback through Java.